{{cookiecutter.module_name|slugify}}/.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Description
+
+* Short description here *
+
+Closes *Add ticket reference here*
+
+# Merge procedure
+
+Don't use the github built-in merge, but the process described [here](https://docs.internal.inmanta.com/topics/tasks/commiting_changes_modules.html)
```

Comment:
This whole file is outdated: see updated version here https://github.com/inmanta/std/blob/master/.github/pull_request_template.md

Reply:
Indeed, lsm needs to be updated